test/BinTestCases.test.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+/* globals describe it */
+const should = require("should");
+const path = require("path");
+const fs = require("fs");
+const spawn = require("child_process").spawn;
+
+function loadOptsFile(optsPath) {
+	// Options file parser from Mocha
+	// https://github.com/mochajs/mocha/blob/2bb2b9fa35818db7a02e5068364b0c417436b1af/bin/options.js#L25-L31
```

Comment:
Opts file parser taken from Mocha code, under MIT. Let me know if that's not ok to do.